j2k/src/org/jetbrains/kotlin/j2k/SpecialFiled.kt

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2010-2016 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jetbrains.kotlin.j2k
+
+import com.intellij.psi.CommonClassNames.*
+import com.intellij.psi.PsiField
+import org.jetbrains.kotlin.builtins.PrimitiveType
+import org.jetbrains.kotlin.j2k.ast.Expression
+import org.jetbrains.kotlin.j2k.ast.Identifier
+import org.jetbrains.kotlin.j2k.ast.QualifiedExpression
+
+enum class SpecialFiled(val qualifiedClassName: String?, val fieldName: String) {
+
+    BYTE_MAX_VALUE(JAVA_LANG_BYTE, "MAX_VALUE") {
```

Comment:
You should not hard-code all possible constants.

Proper way is to check if class mapped to kotlin class, and if mapped and kotlin class has constant with such name and type, then preform convert